.github/kokoro/db-full.sh

```diff
@@ -148,12 +148,14 @@ echo "----------------------------------------"
 	# Pretty HTML file version
 	diff2html --summary=open --file diff.html --format html \
 		-- \
-		--irreversible-delete --find-renames --find-copies --ignore-all-space origin/master
+		--irreversible-delete --find-renames --find-copies \
```

Comment:
This would probably be better....
```suggestion
		--irreversible-delete \
		--find-renames \
		--find-copies \
		--ignore-all-space \
		origin/master || true
```